ios/FluentUI/Drawer/DrawerController.swift

```diff
@@ -446,6 +448,21 @@ open class DrawerController: UIViewController, TokenizedControlInternal {
         self.sourceView = nil
         self.sourceRect = nil
         self.barButtonItem = barButtonItem
+        self.sourceItem = nil
+        self.presentationOrigin = presentationOrigin == -1 ? nil : presentationOrigin
+        self.presentationDirection = presentationDirection
+        self.preferredMaximumExpansionHeight = preferredMaximumHeight
+
+        super.init(nibName: nil, bundle: nil)
+
+        initialize()
+    }
+
+    @objc public init(sourceItem: any UIPopoverPresentationControllerSourceItem, presentationOrigin: CGFloat = -1, presentationDirection: DrawerPresentationDirection, preferredMaximumHeight: CGFloat = -1) {
```

Comment:
I didn't catch this earlier but we need to document this initializer the same way the others are. I'll merge the PR upon that change, thanks!

Reply:
@laminesm Thanks. I refined.